packages/library/src/rpc-default-config.ts

```diff
@@ -3,6 +3,7 @@ import { createSolanaRpcApi } from '@solana/rpc-core';
 import { SolanaJsonRpcIntegerOverflowError } from './rpc-integer-overflow-error';
 
 export const DEFAULT_RPC_CONFIG: Partial<Parameters<typeof createSolanaRpcApi>[0]> = {
+    defaultCommitment: 'confirmed',
```

Comment:
This is the place where the default _actually_ gets configured, and you can see it lives in `library/`